src/nginx.go

```diff
@@ -1,41 +1,92 @@
 package main
 
 import (
+	"fmt"
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
+	"github.com/newrelic/infra-integrations-sdk/data/metric"
+	"github.com/newrelic/infra-integrations-sdk/integration"
 	"github.com/newrelic/infra-integrations-sdk/log"
-	"github.com/newrelic/infra-integrations-sdk/sdk"
+	"net/url"
 )
 
 type argumentList struct {
 	sdk_args.DefaultArgumentList
-	StatusURL  string `default:"http://127.0.0.1/status" help:"NGINX status URL."`
-	ConfigPath string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
+	StatusURL        string `default:"http://127.0.0.1/status" help:"NGINX status URL."`
+	ConfigPath       string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
+	RemoteMonitoring bool   `default:"false" help:"Allows to monitor multiple instances."`
```

Comment:
The help comment may be something confusing, since it would look that simply setting this to "true", multiple instances are set.

Maybe something like: `Identifies the monitored entity as 'remote'. In doubt: set to true`?

Any other suggestion, @ikavanaghNR